### PR TITLE
samplesheets 190821_SN737_0472_BCDPEFACXX_spike.csv

### PR DESCRIPTION
**Pull request by deyazoubi**
could you please check this out

### CODE REVIEW

**Comment by LayalYasin** [comment]
it looks ok to me

File: data/samplesheets/190821_SN737_0472_BCDPEFACXX_spike.csv [hunk 1]
```diff
@@ -0,0 +1 @@
...
6,KB0266_m,,,,C01,AACGTGAT,Keimbahn,set3,KB0266,mother,
6,KB0270_c,,,,D01,CACTTCGA,Keimbahn,set3,KB0270,patient,
7,KB0270_f,,,,E01,GCCAAGAC,Keimbahn,set4,KB0270,father,
7,KB0270_m,,,,F01,GACTAGTA,Keimbahn,set4,KB0270,mother,
7,KB0275_c,,,,G01,ATTGGCTC,Keimbahn,set4,KB0275,patient,
7,KB0275_f,,,,H01,GATGAATC,Keimbahn,set4,KB0275,father,
8,KB0270_f,,,,E01,GCCAAGAC,Keimbahn,set4,KB0270,father,
8,KB0270_m,,,,F01,GACTAGTA,Keimbahn,set4,KB0270,mother,
8,KB0275_c,,,,G01,ATTGGCTC,Keimbahn,set4,KB0275,patient,
8,KB0275_f,,,,H01,GATGAATC,Keimbahn,set4,KB0275,father,
```
Comment (sjanssen2) [inline data/samplesheets/190821_SN737_0472_BCDPEFACXX_spike.csv]: looks like you messed up the line breaks. Everything is in one single line?!

Comment (sjanssen2) [reply data/samplesheets/190821_SN737_0472_BCDPEFACXX_spike.csv]: It seems to be an issue in github. Raw file looks right. I am starting my review now...

**Comment by sjanssen2** [comment]
Operator: I think it is only Kati since the whole flow cell is Keimbahn

File: data/samplesheets/190821_SN737_0472_BCDPEFACXX_spike.csv (hunk 1 above)
Comment (deyazoubi) [reply data/samplesheets/190821_SN737_0472_BCDPEFACXX_spike.csv]: shall I create or add a (line breaks) to  the sample sheet

**Pull request merged**